src/modules/auto-scroll-to-id.js

```diff
@@ -27,8 +27,10 @@
 		}
 		
 		if (!!offsetSelector) {
-			var offsetItem = $(offsetSelector).eq(0);
-			offset = offsetItem.height() * -1;
+			var offsetItem = $(offsetSelector).each(function () {
+				var t = $(this);
+				offset += t.height() * -1;
```

Comment:
The -1 can be remove since we are now using many elements, the negative can be part of the operation.
```suggestion
				offset -= t.height();
```